src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj

```diff
@@ -31,6 +31,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" />
     <PackageReference Include="OpenIddict.AspNetCore" />
+    <PackageReference Include="OpenIddict.Abstractions" />
```

Comment:
The abstractions package is implicitly referenced by the `OpenIddict.AspNetCore` metapackage. Consider removing it.

Reply:
done